packages/roosterjs-content-model-plugins/lib/edit/inputSteps/handleEnterOnParagraph.ts

```diff
@@ -11,7 +11,6 @@ export const handleEnterOnParagraph: DeleteSelectionStep = context => {
 
     if (context.deleteResult == 'notDeleted' && paraIndex >= 0) {
         const newPara = splitParagraph(context.insertPoint);
```

Comment:
So you didn't pass a new format object here, which means the new paragraph will not have any format?

We should create a list of formats that we want to copy, then remove everything else.

Reply:
Which format we need to pass? Maybe we should pass the ones we need to override?

Reply:
Here are the format I think we need to inherit:
- backgroundColor
- direction
- textAlign
- htmlAlign
- lineHeight
- textIndent
- margin* (under `MarginFormat`)
- padding* (under `PaddingFormat`)

And here are the ones that I'd like to remove
- whiteSpace
- isTextIndentApplied
- id
- border* (under `BorderFormat`)